src/main/scala/io/iohk/ethereum/jsonrpc/TestService.scala

```diff
@@ -168,13 +168,11 @@ class TestService(
     SetChainParamsResponse().rightNow
   }
 
-  private def storeGenesisAccountCodes(accounts: Map[String, GenesisAccount]): Unit = {
-    for {
-      code <- accounts.map(pair => pair._2.code).collect { case Some(code) => code }
-    } {
-      blockchain.storeEvmCode(kec256(code), code).commit()
-    }
-  }
+  private def storeGenesisAccountCodes(accounts: Map[String, GenesisAccount]): Unit =
+    accounts
+      .map(pair => pair._2.code)
+      .collect { case Some(code) => code }
```

Comment:
Ok one more nit I missed earlier: you should be able to combine the `map` and `collect` to deconstruct the pair directly.